tutorial/urls.py

```diff
@@ -7,8 +7,8 @@
 schema_view = get_schema_view(title=API_TITLE)
 
 urlpatterns = [
-    url(r'^', include('snippets.urls')),
-    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
-    url(r'^schema/$', schema_view),
-    url(r'^docs/', include_docs_urls(title=API_TITLE, description=API_DESCRIPTION))
+    re_path(r'^', include('snippets.urls')),
+    re_path(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
+    re_path(r'^schema/$', schema_view),
+    re_path(r'^docs/', include_docs_urls(title=API_TITLE, description=API_DESCRIPTION))
```

Comment:
Could we please simplify all the URLs routes to use `path` instead of `re_path`? That should make them more readable.

Reply:
I think it is OK for this PR